http-filter-example/envoy-v2.yaml

```diff
@@ -0,0 +1,33 @@
+admin:
+  access_log_path: /dev/null
+  address:
+    socket_address: { address: {{ ip_loopback_address }}, port_value: 0 }
+
+static_resources:
+  listeners:
+  - name: listener_0
+    address:
+      socket_address: { address: {{ ip_loopback_address }}, port_value: 0 }
```

Comment:
@htuch how do I refer this ip_loopback_address? json also seems to have the same way but not sure how it is resolved. Should I just give "127.0.0.1"?

Reply:
This is substituted by the integration test framework for you. Does it work?

Reply:
it did not work

Reply:
integration framework looks like expecting "envoy-v2.yaml.conf.json" - it is not doing it for the existing test as well. may be be i am doing some thing wrong

Reply:
Try `ntop_ip_loopback_address` here instead of `ip_loopback_address`. In general, `test/config/integration/server_xds*yaml` is a good reference for config fragments that can be consumed by the integration tests.

Reply:
I don't think you should need a YAML file with the new v2 style integration tests in any case. The core bootstrap proto is built programatically for you. Have you tried removing this file and the Bazel BUILD reference to it below?

Reply:
Yes. This is not required. I will nuke this, .json and v1 test case